2024-06-17 04:19:17,560 INFO: Library startup [in D:\alllab\RDExam\app\__init__.py:35]
2024-06-17 04:19:26,859 INFO: Форма добавления книги не прошла валидацию. [in D:\alllab\RDExam\app\routes.py:62]
2024-06-17 04:20:15,361 INFO: Форма добавления книги не прошла валидацию. [in D:\alllab\RDExam\app\routes.py:62]
2024-06-17 04:20:27,903 INFO: Форма добавления рецензии не прошла валидацию. [in D:\alllab\RDExam\app\routes.py:181]
2024-06-17 04:20:32,850 INFO: Рецензия для книги с ID 26 успешно добавлена. [in D:\alllab\RDExam\app\routes.py:173]
2024-06-17 04:20:55,706 INFO: Форма добавления книги не прошла валидацию. [in D:\alllab\RDExam\app\routes.py:62]
2024-06-17 04:24:32,439 INFO: Library startup [in D:\alllab\RDExam\app\__init__.py:35]
2024-06-17 04:25:22,025 INFO: Форма добавления книги прошла валидацию. [in D:\alllab\RDExam\app\routes.py:27]
2024-06-17 04:25:22,126 INFO: Обложка книги сохранена: photo_2024-06-06_03-31-58_2.jpg [in D:\alllab\RDExam\app\routes.py:43]
2024-06-17 04:25:22,536 INFO: Книга 123412 добавлена в базу данных. [in D:\alllab\RDExam\app\routes.py:52]
2024-06-17 04:25:34,414 INFO: Книга с ID 28 обновлена. [in D:\alllab\RDExam\app\routes.py:82]
2024-06-17 04:25:40,042 INFO: Рецензия добавлена для книги с ID 28 пользователем 123456. [in D:\alllab\RDExam\app\routes.py:173]
2024-06-17 04:28:31,837 INFO: Library startup [in D:\alllab\RDExam\app\__init__.py:35]
2024-06-17 04:29:07,981 INFO: Форма добавления книги прошла валидацию. [in D:\alllab\RDExam\app\routes.py:27]
2024-06-17 04:29:08,325 INFO: Обложка книги сохранена: photo_2024-06-06_03-31-59_3.jpg [in D:\alllab\RDExam\app\routes.py:43]
2024-06-17 04:29:08,513 INFO: Книга 1221 добавлена в базу данных. [in D:\alllab\RDExam\app\routes.py:52]
2024-06-17 04:37:05,931 INFO: Library startup [in D:\alllab\RDExam\app\__init__.py:35]
2024-06-17 04:37:36,545 INFO: Форма добавления книги прошла валидацию. [in D:\alllab\RDExam\app\routes.py:27]
2024-06-17 04:37:36,936 INFO: Обложка книги сохранена: photo_2024-06-06_03-28-13.jpg [in D:\alllab\RDExam\app\routes.py:43]
2024-06-17 04:37:37,107 INFO: Книга 1234567 добавлена в базу данных. [in D:\alllab\RDExam\app\routes.py:52]
2024-06-17 04:44:29,521 INFO: Library startup [in D:\alllab\RDExam\app\__init__.py:35]
2024-06-17 04:44:37,135 INFO: Обложка книги удалена: photo_2024-06-06_03-28-13.jpg [in D:\alllab\RDExam\app\routes.py:99]
2024-06-17 04:44:37,523 INFO: Книга с ID 30 удалена. [in D:\alllab\RDExam\app\routes.py:103]
2024-06-17 04:44:42,055 INFO: Обложка книги удалена: photo_2024-06-06_03-31-59_3.jpg [in D:\alllab\RDExam\app\routes.py:99]
2024-06-17 04:44:42,412 INFO: Книга с ID 29 удалена. [in D:\alllab\RDExam\app\routes.py:103]
2024-06-17 04:44:50,879 INFO: Обложка книги удалена: photo_2024-06-06_03-31-58_2.jpg [in D:\alllab\RDExam\app\routes.py:99]
2024-06-17 04:44:51,007 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: {'book_id': None, 'review_id': 12}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:44:55,206 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: {'book_id': None, 'review_id': 12}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:45:01,697 INFO: Обложка книги удалена: photo_2024-06-06_03-28-12_3.jpg [in D:\alllab\RDExam\app\routes.py:99]
2024-06-17 04:45:01,827 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: {'book_id': None, 'review_id': 9}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:45:06,884 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: {'book_id': None, 'review_id': 9}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:45:17,564 INFO: Обложка книги удалена: photo_2024-06-06_03-31-57.jpg [in D:\alllab\RDExam\app\routes.py:99]
2024-06-17 04:45:17,687 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: [{'book_id': None, 'review_id': 7}, {'book_id': None, 'review_id': 10}]]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:45:39,984 INFO: Форма добавления книги прошла валидацию. [in D:\alllab\RDExam\app\routes.py:27]
2024-06-17 04:45:40,012 INFO: Обложка книги сохранена: photo_2024-06-06_03-31-55.jpg [in D:\alllab\RDExam\app\routes.py:43]
2024-06-17 04:45:40,186 INFO: Книга 213456 добавлена в базу данных. [in D:\alllab\RDExam\app\routes.py:52]
2024-06-17 04:46:13,763 INFO: Рецензия добавлена для книги с ID 31 пользователем 123456. [in D:\alllab\RDExam\app\routes.py:173]
2024-06-17 04:46:19,749 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: {'book_id': None, 'review_id': 13}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:46:20,247 ERROR: Ошибка при удалении книги: (pymysql.err.IntegrityError) (1048, "Column 'book_id' cannot be null")
[SQL: UPDATE review SET book_id=%(book_id)s WHERE review.id = %(review_id)s]
[parameters: {'book_id': None, 'review_id': 13}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in D:\alllab\RDExam\app\routes.py:108]
2024-06-17 04:47:06,126 INFO: Форма добавления книги прошла валидацию. [in D:\alllab\RDExam\app\routes.py:27]
2024-06-17 04:47:06,163 INFO: Обложка книги сохранена: photo_2024-06-06_03-28-12_3.jpg [in D:\alllab\RDExam\app\routes.py:43]
2024-06-17 04:47:06,479 INFO: Книга 234567 добавлена в базу данных. [in D:\alllab\RDExam\app\routes.py:52]
2024-06-17 04:47:13,696 INFO: Книга с ID 32 удалена. [in D:\alllab\RDExam\app\routes.py:103]
